.github/workflows/linux-qe.yml

```diff
@@ -0,0 +1,118 @@
+          machine=`tail -n 1 info`
+          echo ${machine##*@} > host
+          echo "root" > username
+          #ssh -o StrictHostKeyChecking=no -i id_rsa root@$host
+     
+      - name: Download linux binary
+        uses: actions/download-artifact@v4
+        with:
+          name: linux-binary
+          run-id: ${{ github.event.workflow_run.id }}
+          github-token: ${{ github.token }}
+
+      - name: Download qe oci image
+        id: download-qe-oci-image-artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: crc-e2e-linux-arm64
+          run-id: ${{ github.event.workflow_run.id }}
+          github-token: ${{ github.token }}
+
+      - name: Correlate
+        env:
+          PULL_SECRET: ${{ secrets.PULL_SECRET }}
+        run: |
+          # Save pull-secret as file
+          echo "${PULL_SECRET}" > pull-secret
+
+      - name: Install CRC on host
+        run: |
+          podman run --rm -d --name crc-linux-install \
+            -e TARGET_HOST=$(cat host) \
+            -e TARGET_HOST_USERNAME=$(cat username) \
+            -e TARGET_HOST_KEY_PATH=/data/id_rsa \
+            -e TARGET_FOLDER=crc-support \
+            -e TARGET_CLEANUP='false' \
+            -e OUTPUT_FOLDER=/data \
+            -e DEBUG='true' \
+            -v ${PWD}:/data:z \
+            -v ${PWD}/crc:/opt/crc-support/crc:z \
+            quay.io/rhqp/crc-support:v0.6-linux crc-support/install.sh crc-support/crc
+          podman logs -f crc-linux-install
+
+      - name: Run CRC e2e test
+        run: |
+          # load image
+          podman load -i crc-e2e-linux-arm64.tar
+          # run
+          cmd="crc-qe/run.sh -junitFilename crc-e2e-junit.xml -targetFolder crc-qe"
+          cmd_microshift="${cmd} -e2eTagExpression '@story_microshift'"
+          cmd_openshift="${cmd} -e2eTagExpression '~@minimal && ~@story_microshift'"
+          podman run --rm -d --name crc-e2e \
+            -e TARGET_HOST=$(cat host) \
+            -e TARGET_HOST_USERNAME=$(cat username) \
+            -e TARGET_HOST_KEY_PATH=/data/id_rsa \
+            -e TARGET_FOLDER=crc-qe \
+            -e TARGET_RESULTS=results \
+            -e OUTPUT_FOLDER=/data \
+            -e DEBUG=true \
+            -v $PWD/pull-secret:/opt/crc/pull-secret:z \
+            -v $PWD:/data:z \
+            quay.io/crcont/crc-e2e:gh-linux-armd64 \
+                ${cmd_openshift} 
+          podman logs -f crc-e2e
+
+      - name: Test Report
+        id: test-report
+        uses: mikepenz/action-junit-report@v4
+        if: always() 
+        with:
+          fail_on_failure: true
+          include_passed: true
+          detailed_summary: true
+          require_tests:  true
+          report_paths: '**/*.xml'
+      
+      - name: Upload e2e results
+        uses: actions/upload-artifact@v4
+        if: always()
+        with:
+          name: linux-e2e-test
+          path: |
+            **/*.xml
+            **/*.results
+            **/*.log
```

Comment:
After reserve, there is no need for decommission the testing farm machine (the target one for arm I mean)